server/mcp_server_hbase/src/mcp_server_hbase/server.py

```diff
@@ -0,0 +1,489 @@
+import os
+import re
+import argparse
+import logging
+from typing import List, Dict, Any, Optional, Annotated
+from mcp.server.fastmcp import FastMCP
+from pydantic import Field
+
+from resource.hbase_resource import HBASESDK
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+mcp_server = FastMCP("hbase_mcp_server", port=int(os.getenv("MCP_SERVER_PORT", "8000")))
```

Comment:
add best practice of streamble http support:
mcp = FastMCP("LAS Dataset Search Server",                    
              host=os.getenv("MCP_SERVER_HOST","0.0.0.0"), # veFaaS health check with external check
              port=int(os.getenv("MCP_SERVER_PORT", "8000")), # your default port
              streamable_http_path=os.getenv("STREAMABLE_HTTP_PATH", "/mcp")), # default streamable-http path
              stateless_http=os.getenv("STATLESS_HTTP", "true").lower() == "true" #support to scale in to 0